### ISSUE TITLE
Columns hidden

### ISSUE THREAD

**Issue by dfujim** [dfujim/SpreadsheetPlagiarism#3]
Joss finds columns A-H hidden in the spreadsheet.

**Comment by dfujim** [comment]
Commit ba526ccfc4ced312a1aa579e6c78b12fd78bd40b explicitly sets columns to visible.

**Comment by dfujim** [comment]
This is apparently an issue with how windows stores column widths. 

"The problem is that for some reason Excel stores column widths in relation to the number of characters whereas everything else is either points or pixels which are convertible"

See this [issue report](https://bitbucket.org/openpyxl/openpyxl/issues/293/column-width-issue).

**Comment by dfujim** [comment]
Also see this [bug report](https://bitbucket.org/openpyxl/openpyxl/issues/947/cant-hide-unhide-column-a)

**Comment by dfujim** [comment]
Column widths print as "None" ([should set to default](https://stackoverflow.com/questions/32855656/column-and-row-dimensions-in-openpyxl-are-always-none))

In multifile_comparer, print_spreadsheet:
```python 
        # adjust column sizes, prevent hiding
        for i,_ in enumerate(sht.columns): 
            sht.column_dimensions[get_column_letter(i+1)].auto_size = True
            sht.column_dimensions[get_column_letter(i+1)].hidden = False
            print(sht.column_dimensions[get_column_letter(i+1)].width)      # this line prints None
```

**Comment by dfujim** [comment]
Joss: I just checked and did an autosize width and it is that the widths are zero. You could set them to a specific size (standard size seems to be 96 pixels).

----

Derek: Fixing the widths is annoying to view in LibreOffice, Ubuntu 18.04.1:

![screenshot from 2018-11-29 14-36-27](https://user-images.githubusercontent.com/7872559/49256562-314ca600-f3e4-11e8-8a99-0729986cb5c1.png)

**Comment by dfujim** [comment]
Note the column widths are set in number of characters. The following appears to set reasonable output. 

```python
# adjust column sizes, prevent hiding
for i,_ in enumerate(sht.columns): 
    sht.column_dimensions[get_column_letter(i+1)].auto_size = False
    sht.column_dimensions[get_column_letter(i+1)].hidden = False
sht.sheet_format.defaultColWidth = 15
```

**Comment by dfujim** [comment]
Resolved in commit 8571cf8
